src/components/tab/tab.ts

```diff
@@ -52,16 +54,7 @@ export default class WaTab extends WebAwesomeElement {
   @watch('disabled')
   handleDisabledChange() {
     this.setAttribute('aria-disabled', this.disabled ? 'true' : 'false');
-  }
-
-  /** Sets focus to the tab. */
-  focus(options?: FocusOptions) {
-    this.tab.focus(options);
-  }
-
-  /** Removes focus from the tab. */
-  blur() {
-    this.tab.blur();
+    this.tabIndex = -1;
```

Comment:
If someone programmatically disables the active tab, the roving tab index is lost so you can't tab into it anymore. Perhaps we should expose a function to tell the Tab Group to reset the tabindex if the active tab gets disabled.

And this will prevent active styles from being applied to disabled tabs.

```suggestion
    this.active = false;    
```

Reply:
🤔 good call...let me play around with it